apps/web/src/components/schedule/DateDropdown/dayPicker.css

```
@@ -0,0 +1,315 @@
+.rdp-range_start .rdp-day_button {
+  background-color: var(--rdp-range_start-date-background-color);
+  color: var(--rdp-range_start-color);
+}
+
+.rdp-range_middle {
+  background-color: var(--rdp-range_middle-background-color);
+}
+
+.rdp-range_middle .rdp-day_button {
+  border-color: transparent;
+  border: unset;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)